content/docs/user-guide/interactivity/robotics/joints-manipulation.md

```diff
@@ -0,0 +1,107 @@
+---
+linkTitle: Joints Manipulation 
+title: Joints Manipulation
+description: Controlling joint systems such as manipulator arms with ROS 2 Gem Open 3D Engine (O3DE).
+toc: true
+weight: 510
+---
+
+## Overview
+
+Manipulating joints is crucial for applications involving robotic arms or other joint systems such as legged locomotion.
+O3DE supports control of joint systems with ROS through packages such as [ros2_controllers](https://github.com/ros-controls/ros2_controllers),
+and through integration with [MoveIt2](https://moveit.ros.org/).
+
+### Supported features
+
+With manipulation components, you can:
+- Publish [JointState](https://docs.ros2.org/latest/api/sensor_msgs/msg/JointState.html) ROS messages automatically with __JointsManipulationComponent__.
+- Control joint systems through [FollowJointTrajectory](https://github.com/ros-controls/control_msgs/blob/humble/control_msgs/action/FollowJointTrajectory.action) ROS action.
+- Control systems based on articulations as well as hinge and prismatic joints.
+- Extend interfaces for developers to implement your own control components through __JointsManipulationRequests__ and __JointsPositionControllerRequests__.
+- Use manipulation components in multi-robot scenarios.
+
+### Limitations
+
+Manipulation components only work with single degree of freedom joints.
+
+### Components and their interactions
+
+Movement of joints is handled through a set of interfaces and components that implement them.
+
+| Interface                            | Components                                                                     | Role                                                                                  |
+|--------------------------------------|--------------------------------------------------------------------------------|---------------------------------------------------------------------------------------|
+| __JointsPositionControllerRequests__ | __JointsArticulationControllerComponent__<br/>__JointsPIDControllerComponent__ | Move joints towards desired positions.                                                |
+| __JointsManipulationRequests__       | __JointsManipulationComponent__                                                | Hold and publish joints state information, relay commands to controllers              |
+| __JointsTrajectoryRequests__         | __JointsTrajectoryComponent__                                                  | Host action server for trajectory commands, control trajectory through goal positions |
```

Comment:
"goal positions" - is "goal" the word commonly used there? What may sound slightly better is "control trajectory through intermediate positions".

Reply:
Changed to "through a sequence of positions"